src/components/Banner/index.tsx

```diff
@@ -12,11 +12,16 @@ type BannerProps = {
   ctaLink: string;
   startDate?: string;
   endDate?: string;
-  releaseBanner?: boolean;
+  releaseBanner?: boolean;  // Indicates if this is a Release banner (priority banner)
 };
 
 // -------------------------------------------------------
 // NOTE: Add your list of current banners here
+// - Each banner can have optional startDate and endDate in ISO format
+// - Banners with dates are only valid within that range
+// - Banners are always displayed randomly
+// - Banners marked as releaseBanner will be prioritized if within date range
+// - Do not set releaseBanner to true for non-release banners
```

Comment:
Are all these fields required? If not, what do they default to when unspecified?